Design assemblies should only target v4.5

### Description
The .Design assemblies are targeting .NET 4.7, but if you use Xamarin.Forms in WPF and target an older version (4.5-4.7) you'll get the following warnings:
```
2>------ Build started: Project: FormsTestApp.WPF, Configuration: Debug Any CPU ------
2>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(2052,5): warning MSB3274: The primary reference "Xamarin.Forms.Core.Design, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL" could not be resolved because it was built against the ".NETFramework,Version=v4.7" framework. This is a higher version than the currently targeted framework ".NETFramework,Version=v4.6.1".
2>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(2052,5): warning MSB3274: The primary reference "Xamarin.Forms.Xaml.Design, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL" could not be resolved because it was built against the ".NETFramework,Version=v4.7" framework. This is a higher version than the currently targeted framework ".NETFramework,Version=v4.6.1".
2>  FormsTestApp.WPF -> c:\apps\dotnet\TestApps\FormsTestApp.WPF\bin\Debug\FormsTestApp.WPF.exe
========== Build: 1 succeeded, 0 failed, 1 up-to-date, 1 skipped ==========
```
### Steps to Reproduce

1. Create a WPF project targeting 4.6.1
2. Reference `Xamarin.Forms.Platform.WPF.3.0.0.296286-pre2` and the dependencies which only requires 4.5.0
3. Build, and observe the above warnings

### Expected Behavior
No warning is generated.

### Actual Behavior
Build warnings

### Basic Information

- Version with issue: 3.0.0.296286-pre2
- Last known good version:
- IDE: VS2017 (15.5.2)
- Platform Target Frameworks: <!-- All that apply -->
  -net461
- Android Support Library Version: <!-- if applicable -->
- Nuget Packages:
```xml
    <PackageReference Include="Xamarin.Forms">
      <Version>3.0.0.296286-pre2</Version>
    </PackageReference>
    <PackageReference Include="Xamarin.Forms.Platform.WPF">
      <Version>3.0.0.296286-pre2</Version>
    </PackageReference>
```
- Affected Devices:

### Screenshots
![image](https://user-images.githubusercontent.com/1378165/38160384-b6f6d3bc-3471-11e8-8d4f-66ce36cd8744.png)

--- a/Xamarin.Forms.Core.Design/Xamarin.Forms.Core.Design.csproj
+++ b/Xamarin.Forms.Core.Design/Xamarin.Forms.Core.Design.csproj
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Xamarin.Forms.Core.Design</RootNamespace>
     <AssemblyName>Xamarin.Forms.Core.Design</AssemblyName>
-    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>

--- a/Xamarin.Forms.Maps.Design/Xamarin.Forms.Maps.Design.csproj
+++ b/Xamarin.Forms.Maps.Design/Xamarin.Forms.Maps.Design.csproj
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Xamarin.Forms.Maps.Design</RootNamespace>
     <AssemblyName>Xamarin.Forms.Maps.Design</AssemblyName>
-    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
     <NuGetPackageImportStamp>

